eservice/bin/ss-status.sh

```diff
@@ -14,7 +14,31 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-PLIST=$(pgrep -u ${USER:-$(whoami)} -f sservice)
+F_SERVICEHOME="$( cd -P "$( dirname ${BASH_SOURCE[0]} )/.." && pwd )"
+source ${F_SERVICEHOME}/bin/common.sh
+
+F_USAGE='-b|--base name'
+F_BASENAME='sservice'
+
+# -----------------------------------------------------------------
+# Process command line arguments
+# -----------------------------------------------------------------
+TEMP=`getopt -o b:h --long base:,help \
+     -n 'stop.sh' -- "$@"`
```

Comment:
1) again, I don't think the idea here is to shut down the services if there is any error
2) stop.sh should be the name of the program called by getopt ... but where is it defined?

Reply:
see above